src/conditional-flow/multiple-conditions.js

```diff
@@ -2,6 +2,13 @@
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
 function isInRange (num, lower, upper) {
+  if (num >= lower && num <= upper) {
```

Comment:
This can be simplified. `num >= lower && num <= upper` creates a boolean value representing if the num value is greater than or equal to lower AND is less than or equal to upper. So we can just return that value like so:
```js
return num >= lower && num <= upper
```